assets/dist/styles.css

```diff
@@ -0,0 +1,340 @@
+:root {
```

Comment:
This is quite a lot of "custom" css. Have you thought about using a framework such as bootstrap?
This way you only need to specify custom components and have mobile support for free

Reply:
Implemented bootstrap and reduce custom css